source/runbooks/revoke-network-access.html.md.erb

```diff
@@ -0,0 +1,117 @@
+
+> Additionally we are only making the changes manually to prevent the code being updated to the detriment of subsequent repairs. Assuming the issue is resolved we would need to rerun the original terraform code to reset everything to it's previous state. Any changes made to the code could cause us issues in being able to do this. It is likely that when applying the code there will be an warning shown stating that some changes have been made outside terraform. This is not something to worry about as all access will be reset to the previous state.
+
+## Preventing access to a single application
+
+The objective here is to stop any access to a specific single application on the servers. The most appropriate way is to remove the items from the security group. The details are listed below
+
+1. Connect to the server containing the storage group and go to the VPC option. 
+2. Pick Security->Security group
+3. Select the impacted security group.
+4. You can either delete the entire security group using Actions->Delete security groups. For speed you can do it this way but it isn't really recommended.
+5. Go into **BOTH** Edit inbound rules and Edit outbound rules.
+6. You can note these but they should be rebuilt by terraform code. "Delete" each rule listed. For some security groups there may be a long list of items to delete. **NOTE** They are removed immediately with no option to go back. If you want to go back you can select the group again from the list above.
+7. When you are happy click The Save rules button
+
+## Stopping access from the internet to applications on MP
+
+This work is carried out from the core-vpc-(environment) environment e.g. core-vpc-production
+
+If you know which business unit the application under attack is in  you can disconnect the VPC from the internet by following these instructions.
+
+1. Go to VPC->Internet gateways
+2. Select the gateway (example is hmpps-preproduction-internet-gateway)
+3. Note the Internet gateway ID and the VPC ID
+4. Click on Actions, Detach from VPC
+5. If you are sure this is correct click on the Detach internet gateway button
+
+The selected internet gateway will be detached from the VPC.
+
+This change will affect all applications in the VPC that you have selected. No applications in the VPC will be accessible from the internet. 
+
+## Stopping access to the internet for applications on MP
+
+Due to the way Network Firewall applies rules in AWS (see [here](https://docs.aws.amazon.com/network-firewall/latest/developerguide/suricata-rule-evaluation-order.html) ), this section only covers changes to the VPC NACL. 
+
+This work is carried out from the core-vpc-(environment) environment e.g. core-vpc-production.
+
+The process needs to be done for all 3 NACLs listed. An example is
```

Comment:
There are 5 NACLs sets per each business unit's environment and there are 3 subnets associated with each NACL set. This needs rewriting to be accurate.

Reply:
you can probably skip the transit gateway subnets, but need all: data, private, protected and public

Reply:
I don't `protected` subnets are going to be relevant in this case as they never hold customer infrastructure and should only host VPC endpoints.

Reply:
This has been reduced to the 3 (data, private, public) as per David's suggestion.

Reply:
true, although is there anything stopping users from hosting infrastructure there?

Reply:
also am I OK to assume that nothing can be deployed in the transit subnet if the TGW attachment is in place? or should we worry about that subnet too?

Reply:
The TGW subnets are not shared across the whole system. We should not have issues. A bad actor would also need access to the code and be able to amend and deploy it.

Reply:
After having a discussion in slack, this is now to extend the list of NACLs, for `core-*` accounts

Reply:
To include the protected network this has been included in the list so there are now 4 shown. 
The transit gateway is shown in 4 other locations so these will be listed too.